cirq-google/cirq_google/engine/engine_processor_test.py

```diff
@@ -179,6 +179,32 @@ def _to_timestamp(json_string):
 )
 
 
+class FakeEngineClient(engine_client.EngineClient):
+    """Fake engine client for testing."""
+
+    def __init__(self):
+        super().__init__()
+        self._processor = quantum.QuantumProcessor()
+
+    async def get_processor_async(
```

Comment:
This doesn't do anything async and doesn't seem to be called by the system under test. Is it needed? It seems like `getProcessor()` could just return the value instead.

Reply:
The fake needed to match the method signature of its parent class, otherwise errors arose or it would default to the super class. My initial attempt was to do what you described but that caused errors. Regardless, this chunk was removed in the update in favor of a mock client. :)